src/server/object_bench_test.go

```diff
@@ -0,0 +1,96 @@
+package server
```

Comment:
I was able to get about 37 MB/s of upload speed, 40 MB/s download when the cache was empty and 120 MB/s down when the cache was already loaded. All this was from multiple clients running on a single pod.